libsolidity/analysis/TypeChecker.cpp

```diff
@@ -45,9 +45,10 @@ using namespace dev;
 using namespace langutil;
 using namespace dev::solidity;
 
-namespace
+namespace dev
```

Comment:
We usually don't use scoping namespaces in cpp files

Reply:
It wouldn't compile otherwise, something about ambiguous call/definition. Using static will resolve this though.